test/src/lib/QueryParser.js

```diff
@@ -89,6 +89,7 @@ describe('Query parser', function() {
       var seen = {};
       for ( var i = 0; i < actual.array.length; i++ ) {
         seen[actual.array[i].id] = true;
+        if ( ! ( expected.indexOf(actual.array[i].id) >-1 ) ) debugger;
```

Comment:
This looks like a statement you included for debugging purposes. Can it be removed?